eng/pipelines/ci.yml

```diff
@@ -1,9 +1,3 @@
-parameters:
```

Comment:
Why are we removing SkipSigning?   It was an easy way to trim 20-30 min from builds that don't deliver any artifacts

Reply:
I didn't see any good usages to skip signing. We are either releasing or not and if we are releasing then we need to sign.

If anything we could control the release but not the signing itself.